opensrp-fp/src/main/java/org/smartregister/fp/common/util/Utils.java

```diff
@@ -200,44 +206,80 @@ public static int getProfileImageResourceIdentifier() {
      */
     public static void proceedToContact(String baseEntityId, HashMap<String, String> personObjectClient, Context context) {
         try {
+            /*baseEntityId = "ba9b0b5c-4453-4fc3-8d62-f13fbaf0a342";
```

Comment:
Code clean up >> Do you intend to use the commented code later? You can clean this up if not